src/Compiler/CompilerUtils.cpp

```diff
@@ -1050,11 +1050,13 @@ void emitOutput(mlir::OwningOpRef<ModuleOp> &module, mlir::MLIRContext &context,
 int compileModule(mlir::OwningOpRef<ModuleOp> &module,
     mlir::MLIRContext &context, std::string outputBaseName,
     EmissionTargetType emissionTarget) {
+  extern void InitAccelerators();    
   setupModule(module, context, outputBaseName);
 
   mlir::PassManager pm(&context, mlir::OpPassManager::Nesting::Implicit);
   // Initialize accelerator if required
   if (acceleratorTarget.compare("") != 0) {
```

Comment:
`acceleratorTarget` should be defined in this file so that it shows up in the correct section in the output of `onnx-mlir --help`.  

Propose to use `-maccel` as the option to use to select the accelerator target, as in: 
```
static llvm::cl::list<accel::Accelerator::Kind> maccel("maccel",
llvm::cl::desc("Specify an accelerator to generate code for"),
    llvm::cl::values(clEnumValN(
        accel::Accelerator::Kind::NNPA, "NNPA", "IBM Telum processor")),
    llvm::cl::cat(OnnxMlirOptions), llvm::cl::ValueRequired);
```
The `Accelerator` class would have to define a strongly typed enum as in: 
```
namespace onnx_mlir {
namespace accel {

class Accelerator {
public:
  /// Kinds of accelerators.
  enum class Kind {
    NNPA, // IBM Telum coprocessor
  };
  . . .
```
Doing so would help avoid the string comparison and instead of: 
```
if (acceleratorTarget.compare("") != 0)
```
we can then write:
```
if (!maccel.empty()) {
```